### PR TITLE
Add “whats it like" section to engineer and reliability engineer

### PR DESCRIPTION
**Pull request by lukemorton**

### CODE REVIEW

File: roles/as_code/lib/engineer.rb [hunk 1]
```diff
@@ -29,6 +29,20 @@
     - Infrastructure as code tools like Ansible, Chef, Puppet, and/or Saltstack
     - Strong delivery skills: a huge drive to get something shipped
 
+    ## What is it like to work at Made Tech?
+
+    We are a sanctuary for those wanting to hone their skills alongside like-minded learners. After joining our team it is common for new starters to comment on how much they’re learning and how much they enjoy the fact they are surrounded by people they can learn from. This includes our most senior hires.
+
+    The biggest thing you’ll take away from our culture after spending the day with us would be that we practice continuous improvement at every level. Everyone has peer-based one-to-ones every 2 weeks, along with monthly one-to-ones with a director. Teams have fortnightly retrospectives. We also hold a company wide retrospective fortnightly. We discuss our problems out in the open, and rather than punish failure we band together and to find solutions.
```
Comment (zuren) [inline roles/as_code/lib/engineer.rb]: "band together _to find_ solutions"

File: roles/as_code/lib/engineer.rb [hunk 2]
```diff
@@ -29,6 +29,20 @@
...
+
+    Other notable things:
+
+    - Every Friday afternoon is dedicated to learning new skills
+    - Everyone is encouraged to write blog posts regularly
+    - Our handbook is open sourced
+    - We are vegan and non-drinker friendly as well as meat-eater and drinker friendly
```
Comment (zuren) [inline roles/as_code/lib/engineer.rb]: nice!

**Comment by zuren** [comment]
Not possible to share the text between the roles?

**Comment by lukemorton** [comment]
Not yet :P

**Pull request merged**